cypress/components/action-popover/action-popover.cy.tsx

```diff
@@ -362,6 +372,7 @@ context("Test for ActionPopover component", () => {
       actionPopover().click();
       cy.readFile(downloadedFilename, "binary", {
         timeout: 15000,
+        // eslint-disable-next-line jest/valid-expect
       }).should((buffer) => expect(buffer.length).to.be.gt(100));
```

Comment:
suggestion: I believe we can avoid this ignore if we correct the syntax for the expect

```suggestion
      }).should((buffer) => {
          expect(buffer.length).to.be.greaterThan(100);
      });
```

Reply:
This suggestion will not fix the lint error, so we will stick with the original `eslint-disable-next-line` comment

Reply:
I changed the code to something slightly different which did fix the lint error, so pushing the changes back :)